FRDeviceBinding/FRDeviceBinding/Sources/DeviceBindingAuthenticators.swift

```diff
@@ -119,10 +122,13 @@ extension DeviceAuthenticator {
         //create header
         var header = JWSHeader(algorithm: algorithm)
         header.kid = userKey.kid
-        header.typ = "JWS"
+        header.typ = DBConstants.JWS
         
         //create payload
-        let params: [String: Any] = ["sub": userKey.userId, "challenge": challenge, "exp": (Int(expiration.timeIntervalSince1970))]
+        var params: [String: Any] = [DBConstants.sub: userKey.userId, DBConstants.challenge: challenge, DBConstants.exp: (Int(expiration.timeIntervalSince1970))]
+        if let bundleIdentifier = Bundle.main.bundleIdentifier {
```

Comment:
@Rodrigo, it's only mandatory if it's set in the node under "Application ID" field. 

In other cases in the SDK (like in WebAuthnAuthenticationCallback, WebAuthnRegistrationCallback, KeychainManager) we use a default value if Bundle.main.bundleIdentifier is nil. Should we do the same here? 

@witrisna what's your thoughts on this? In case this is nil,  should we leave it empty, use default value or fail the binding/signing?

Reply:
In most scenario this  Bundle.main.bundleIdentifier value cannot be nil , in the situation of nil , If the application id has a value and the app sending nil , the developer should be able to fix that in the application. but this is a extra step in them to make sure all good.  we should either add this scenario in the docs ? . the only thing we need to worry here is platform i think because apple and android can also share the same application and bundleid